src/lib.rs

```diff
@@ -161,9 +164,7 @@ mod contract {
     /// Call method on the EVM contract.
     #[no_mangle]
     pub extern "C" fn call() {
-        // TODO: Borsh input pattern is so common here. It worth writing sdk::read_input_borsh().
-        let input = sdk::read_input();
-        let args = FunctionCallArgs::try_from_slice(&input).sdk_expect("ERR_ARG_PARSE");
+        let args: FunctionCallArgs = sdk::read_input_borsh().sdk_unwrap();
```

Comment:
Just a general note, I think I was implying the `sdk_unwrap`s to be not really used, unless we are using it for testing or some other purposes. For production, they should always return something clear that we can follow up on.

Reply:
`read_input_borsh` returns a `Result`, so `sdk_unwrap` will use the `AsRef[u8]` implementation on the error type, which returns `ERR_ARG_PARSE` as it did previously. We can change the error message to mention borsh if you think it should be more clear.